portal-frontend/src/app/shared/header/header.component.html

```diff
@@ -1,25 +1,25 @@
 <header>
   <div class="title-wrapper">
-
     <div class="title">
-      <img class="logo" src="assets/logo.png" alt="ALC Portal" />
```

Comment:
is logo.png used anywhere? If no lets delete it from repo

Reply:
I think it's being used in the email template